bignum_kilic_red.go

```diff
@@ -0,0 +1,116 @@
+// +build bignum_kilic_red
```

Comment:
Thanks for the demo, testing different bignum implementations against each other is doable with the build-options. But this is a little bit too much copy-paste of the other kilic BLS file. We could either put the two `norm` function variants in separate files (and only switch that on/off with a build tag), or we just improve the existing kilic bignum file.